Package.swift

```diff
@@ -0,0 +1,22 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "cl-openssl",
+    products: [
+        .library(
+            name: "cl-openssl",
+            targets: ["openssl"]),
+    ],
+    dependencies: [],
+    targets: [
+        .binaryTarget(name: "openssl",
+                      // update version in URL path
+                      url:"https://github.com/julepka/openssl-apple/releases/download/1.1.1080202/openssl-static-xcframework.zip",
```

Comment:
extra zero here.

current version is `v1.1.10802`, next version should be (semver) `1.1.10803`

Reply:
Ok, so the link will be https://github.com/cossacklabs/openssl-apple/releases/download/1.1.10803/openssl-static-xcframework.zip

Reply:
By the way, while we're here, maybe bump OpenSSL to 1.1.1j – released later as 1.1.10901?

Reply:
@ilammy good point, will do

Reply:
@ilammy, after discussion with @vixentael we think that it will be better to release the SPM update and then release the updated OpenSSL as two separate releases. So, the users have a workable version of SPM if the new OpenSSL version breaks something.